client/src/main/java/com/linecorp/decaton/client/DecatonClientImpl.java

```diff
@@ -40,11 +44,12 @@
                       String instanceId,
                       Properties producerConfig,
                       KafkaProducerSupplier producerSupplier,
-                      Supplier<Long> timestampSupplier) {
+                      Supplier<Long> timestampSupplier,
+                      Tracing tracing) {
         this.serializer = serializer;
         this.applicationId = applicationId;
         this.instanceId = instanceId;
-        producer = new DecatonTaskProducer(topic, producerConfig, producerSupplier);
+        producer = new DecatonTaskProducer(topic, producerConfig, producerSupplier, tracing, format("{}:{}",applicationId,instanceId));
```

Comment:
I think using `+` to concatenate strings is straightforward enough here?

Reply:
ok, and follow @kawamuray 's suggestions , here may be  changed in next commit.